recipe/patch_yaml/arcticdb.yaml

```diff
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=../patch_yaml_model.json
+# Use `libcurl<8.17` dependency for all ArcticDB builds up to 6.3.1 (inclusive).
+# See: https://github.com/man-group/ArcticDB/pull/2746
+if:
+  name: arcticdb
+  timestamp_lt: 1763539708000  # 2025-11-19
```

Comment:
Please first add this cap to the arcticdb feedstock. A timestamp here is useless if someone merges another PR on the feedstock that reintroduces a `libcurl <9` bound.

Reply:
This is being done in the next PR on the feedstock (see https://github.com/conda-forge/arcticdb-feedstock/pull/526#discussion_r2541029761) for the `main` branch, and this has been done for the `rc` branch via https://github.com/conda-forge/arcticdb-feedstock/pull/525.

Do you require it to be done via another, dedicated PR?